server/app/services/export/enums/ApiPathSegment.java

```diff
@@ -6,7 +6,9 @@
  * <p>This enum should be treated as append-only.
  */
 public enum ApiPathSegment {
-  QUESTION_TYPE,
-  CURRENCY_DOLLARS,
-  ENTITIES;
+  QUESTION_TYPE, // the name of the field that indicates the question type
```

Comment:
**nitpick (non-blocking):** Consider aligning comments for ease of scanning

Reply:
Unfortunately `bin/fmt` unaligns them, but I did fix the indentation of the comment that wraps.